packages/cli/src/Server.ts

```diff
@@ -392,6 +395,18 @@ class Server extends AbstractServer {
 				postHog,
 			}),
 		];
+
+		if (isLdapEnabled()) {
```

Comment:
this would mean if ldap is enabled at a later point, it would require a restart of the instance (and vice versa) - is that ok?